lib/message/decrypt.js

```diff
@@ -50,7 +50,7 @@ export async function decryptMessage(options) {
 }
 
 // Backwards-compatible decrypt message function
-// 'message' option must be a string!
+// `options.message` must be a string, to properly handle legacy messages (and avoid misusing this function)
```

Comment:
As written, it kind of sounds like it only needs to be a string if you care about legacy messages, which is true - but I would rather change that and remove the checks, so that this function always throws for other types, so that we enforce this rather than write a comment about it.
```suggestion
// `options.message` must be a string
```